github/dependabot.md

```diff
@@ -24,30 +24,59 @@ For instance, [here are some vulnerabilities found in the k8s.io libraries we us
 
 ## Configuration
 
-There are two ways to configure Dependabot:
+Dependabot updates can be enabled for a repository in the "Settings & analysis" tab of the GitHub settings UI.
+To enable updates, simply click the "Enable" button as shown in the image below.
+![Dependabot enable options in GitHub settings UI](dependabot/dependabot-enable.png)
 
-1. Through [the web UI](https://app.dependabot.com/accounts/giantswarm/)
-2. Using [configuration files](https://dependabot.com/docs/config-file/) (preferred)
+To further configure how Dependabot performs updates in your repository, use a configuration file.
 
-Configuration files are preferred because they can be more easily managed per-project and per-branch directly in the repository,
-and expose more configuration options than the web UI.
+For `Go` projects, you should generate one using `devctl gen dependabot` (see [`devctl`](https://github.com/giantswarm/devctl) for usage).
+If needed, you can then modify this file to control the update frequency or adjust dependency preferences.
+However, keep in mind that re-generating with `devctl` will overwrite any such customizations.
```

Comment:
I would strongly suggest skipping this part. We should adapt the generating code if we find some edge cases. If this is impossible then as last resort we should do manual modifications. But let's find what are those edge cases first.

```suggestion
```